SDWebImage/SDWebImageImageIOCoder.m

```diff
@@ -74,6 +74,9 @@ - (BOOL)canDecodeFromData:(nullable NSData *)data {
         case SDImageFormatHEIC:
             // Check HEIC decoding compatibility
             return [[self class] canDecodeFromHEICFormat];
+        case SDImageFormatHEIF:
```

Comment:
Actually...Do we need this check ? By default we return `YES`, and this `SDWebImageIOCoder` should contains the lowest priority in the coders. So check the HEIC/HEIF decoding possibility seems useless. I forget the reason why we use this check...😅

Reply:
I think it's more clear if it's explicit.